README.md

```diff
@@ -1,65 +1,89 @@
+    ```
 
-Alternatively, use [`npx install-peerdeps`](https://www.npmjs.com/package/install-peerdeps) as a shortcut to install the packages for you.
+   Then follow the [JavaScript configuration](#javascript-configuration) instructions.
+- For TypeScript run:
+
+    ```bash
+    npm install @ni/eslint-config-typescript
+    ```
+
+    Then follow the [TypeScript configuration](#typescript-configuration) instructions.
+- For Angular run:
+
+    ```bash
+    npm install @ni/eslint-config-angular
+    ```
+
+    Then follow the [Angular configuration](#angular-configuration) instructions.
+
+If you are using npm 6 or lower, use the following instructions to manually install peer dependencies:
+
+Use [`npm view`](https://docs.npmjs.com/cli/view.html) to list the correct versions of each peer package to install yourself. For example, with a JavaScript project run:
 
 ```bash
-npx install-peerdeps --dev @ni/eslint-config
+npm view @ni/eslint-config-javascript peerDependencies
 ```
 
-### Angular Installation
+Alternatively, use [`npx install-peerdeps`](https://www.npmjs.com/package/install-peerdeps) as a shortcut to install the peer packages for you. For example, with a JavaScript project run:
 
-Follow the [instructions below](#angular) to use a schematic to install dependencies.
+```bash
+npx install-peerdeps --dev @ni/eslint-config-javascript
+```
 
 ## Configuration
 
-### JavaScript
+After installing the lint configuration packages, follow the configuration instructions for your project language:
+
+### JavaScript configuration
 
-Extend `@ni` in the [ESLint configuration](https://eslint.org/docs/user-guide/configuring/configuration-files#configuration-file-formats).
+Extend `@ni/eslint-config-javascript` in the [ESLint configuration](https://eslint.org/docs/user-guide/configuring/configuration-files#configuration-file-formats).
 
 ```js
 {
-    extends: '@ni'
+    extends: '@ni/eslint-config-javascript'
 }
 ```
 
-### TypeScript
+### TypeScript configuration
 
-Extend `@ni/eslint-config/typescript` and `@ni/eslint-config/typescript-requiring-type-checking` in the [ESLint configuration](https://eslint.org/docs/user-guide/configuring/configuration-files#configuration-file-formats). Set the `parserOptions.project` configuration to the project's TypeScript configuration.
+Extend `@ni/eslint-config-typescript` and `@ni/eslint-config-typescript/requiring-type-checking` in the [ESLint configuration](https://eslint.org/docs/user-guide/configuring/configuration-files#configuration-file-formats). Set the `parserOptions.project` configuration to the project's TypeScript configuration.
 
 ```js
 {
     extends: [
-        '@ni/eslint-config/typescript',
-        '@ni/eslint-config/typescript-requiring-type-checking'
+        '@ni/eslint-config-typescript',
+        '@ni/eslint-config-typescript/requiring-type-checking'
     ],
     parserOptions: {
         project: 'tsconfig.json'
     }
 }
 ```
 
-### Angular
+### Angular configuration
```

Comment:
@TrevorKarjanis if you can in particular review the Angular section, I tried to capture the updates to the NI provided configs like we discussed.